pkg/kubelet/api/v1alpha1/runtime/api.proto

```diff
@@ -676,3 +679,17 @@ message RemoveImageRequest {
 }
 
 message RemoveImageResponse {}
+
+message NetworkConfig {
+    optional string podCIDR = 1;
```

Comment:
@squeed Currently, this is a one time update when podCIDR become available. 
Later if we support the concept of multiple networks, then we need to figure out if we want to pass that information at RunPodSandbox.